src/openrct2/scenario/Scenario.cpp

```diff
@@ -944,6 +942,19 @@ ObjectiveStatus Objective::CheckMonthlyFoodIncome() const
     return ObjectiveStatus::Undecided;
 }
 
+bool Objective::AllowEarlyCompletion() const
```

Comment:
Doesn't belong in the objective struct. Please turn this into a regular function.